src/main.py

```diff
@@ -110,13 +118,40 @@ async def on_message(self, message):
             await message.channel.send(f'{", ".join(games)} removed from {author}\'s record')
         elif cmd == '$list':
             user = Player(author)
-            msg = f'{", ".join(user.get_games())}'
-            if msg == '':
-                msg = "No games in library."
-            await message.channel.send(msg)
+            out_msg = f'{", ".join(user.get_games())}'
+            if out_msg == '':
+                out_msg = "No games in library."
+            await message.channel.send(out_msg)
+        elif cmd == '$suggest':
+            all_games = self.get_games_guild(message.guild)
+            if msg.isdigit():   
+                out_msg = self.suggest_game(message.guild, all_games, int(msg))
+            else: 
+                out_msg = "Selected channel is not a voice channel or spelled incorrectly. Try again."
+                for channel in message.guild.channels:
+                    if msg == channel.name and channel.type == discord.ChannelType.voice:
+                        out_msg = self.suggest_game_for_channel(channel, all_games)
+            if out_msg == []:
```

Comment:
If you update the above such that out_msg is always a string and never an arary, you can just change it to this

```suggestion
            if out_msg is None:
```